packages/block-library/src/calendar/index.php

```diff
@@ -13,7 +13,7 @@
  * @return string Returns the block content.
  */
 function render_block_core_calendar( $attributes ) {
-	global $monthnum, $year, $post;
+	global $monthnum, $year;
```

Comment:
Are there lint options available to make it so this sort of error is impossible to introduce?

Reply:
It looks like there are some tools created by the community that allows us to do that https://github.com/sirbrillig/phpcs-variable-analysis.

Reply:
Related discussion https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1221.